README.md

```diff
@@ -46,8 +46,8 @@ This repository is just that: an ever-growing collection of examples
 for the most common UNIX, Linux, macOS, SunOS and Windows command-line tools.
 
 ## How do I use it?
-
-A popular and convenient way to access these pages on your computer
+If you just want to browse without installing anything, check out the [PDF version.](https://github.com/tldr-pages/tldr-pages.github.io/blob/master/assets/tldr-book.pdf)
```

Comment:
It looks like you removed the newline that usually comes after headers...

```suggestion

If you just want to browse without installing anything, check out the [PDF version.](https://github.com/tldr-pages/tldr-pages.github.io/blob/master/assets/tldr-book.pdf)
```

Reply:
@older-pack to apply that suggestion:
1. Click "Commit suggestion"
2. Click "Commit changes"

Reply:
Sorry about that, I'm in over my head! This is my first git pull request.